src/main/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocument.java

```diff
@@ -923,6 +924,48 @@ protected boolean isCampusReviewRequired() {
 
         return false;
     }
+    
+    @Override
+    protected boolean isTaxReviewRequired() {
```

Comment:
Since the bulk of this method is the same as the one it overrides, it would be nice to eliminate the duplication and call super.isTaxReviewRequired(), however that looked like it might be difficult based on my quick review. Given that, I'm not asking for this to be changed, but I wanted to bring it up so others can think along the same lines (avoid duplicating code from overridden methods if possible) and just in case you can see a better way to do that here than I can.